tester/check_replacer.js

```diff
@@ -178,26 +196,26 @@ describe(${args[0]}, ():i32 => {
       let testCounter = 0;
       for (let iValue = 0; iValue < values.length; testCounter++) {
         // Destructuring assignment unpacks returned object values into distinct values.
-        ({got: gotExpr, iValue} = hydrateGot(gotTemplate, values, iValue));
+        ({ instanciation: gotExpr, iValue } = hydrateTemplate(gotTemplate, values, iValue));
+        ({ instanciation: expectExpr, iValue } = hydrateTemplate(expectedTemplate, values, iValue));
 
-        ({test: testExpr, iValue} = generateTest(
-            testCounter, // use test counter as test name
-            gotExpr,
-            ifExpr, needWant,
-            onFailure == 'onFailure.Continue',
-            values, iValue));
+        const test = generateTest(
+          testCounter.toString(), // use test counter as test name
+          gotExpr.replace(/['`]/g, ''), expectExpr.replace(/['`]/g, ''),
+          ifExpr, needWant,
+          onFailure == 'onFailure.Continue');
 
-        expr+= testExpr;
+        expr += test.replace(/\n/g, '\n  '); // adds static indentation
       }
 
-      expr += `return TestResult.Success;\n});\n`;
+      expr += `\n  return TestResult.Success;\n});\n`;
 
       // magic function define at parent level that will:
       // - remove the code used here from the initial file content
       // - create a new file with the generated tests
       this.addUpdate({
         begin: node.range.start,
-        end: node.range.end+2, // +1 to include the trailing semicolon and new line (;\n)
+        end: node.range.end + 2, // +1 to include the trailing semicolon and new line (;\n)
```

Comment:
Isn't it supposed to be `+2` in that comment ?

```suggestion
        end: node.range.end + 2, // +2 to include the trailing semicolon and new line (;\n)
```

Reply:
fixes